docs/conversion/sakai_23_oracle_conversion.sql

```diff
@@ -23,3 +23,22 @@ ALTER TABLE rbc_rating ADD order_index NUMBER(1,0) NULL;
 UPDATE rbc_rating r, rbc_criterion_ratings cr SET r.criterion_id = cr.rbc_criterion_id, r.order_index = cr.order_index WHERE cr.ratings_id = r.id;
 UPDATE rbc_criterion c, rbc_rubric_criterions rc SET c.rubric_id = rc.rbc_rubric_id, c.order_index = rc.order_index WHERE rc.criterions_id = c.id;
 -- END SAK-46178
+
+-- SAK-46257
+ALTER TABLE CONV_POSTS ADD DEPTH NUMBER(2, 0) DEFAULT NULL;
+ALTER TABLE CONV_TOPIC_STATUS ADD POSTED CHAR(1) DEFAULT NULL;
```

Comment:
Is this mean to represent a boolean true/false/0/1? If so it's more appropriate to use `NUMBER(1,0)`